baseplate/lib/metrics.py

```diff
@@ -302,6 +304,9 @@ def send(self, elapsed: float) -> None:
 
```

Comment:
i think we'd want `send()` to take a sample_rate as well. 

```python
def stop(self):
    ...
    self.send(elapsed, sample_rate=self.sample_rate)
    ...

def send(self, elapsed: float, sample_rate: float) -> None:
    ...
    if sample_rate < 1.0:  # note: _not_ self.sample_rate
```

Reply:
👍